cocos/scripting/js-bindings/manual/ScriptingCore.cpp

```diff
@@ -1053,7 +1070,6 @@ void ScriptingCore::releaseScriptObject(cocos2d::Ref* owner, cocos2d::Ref* targe
 void ScriptingCore::unrootScriptObject(cocos2d::Ref* target)
 {
     JS::RootedObject global(_cx, _global->get());
-    JSAutoCompartment ac(_cx, global);
```

Comment:
No need to invoke `JSAutoCompartment` every where since we use JS_EnterCompartment/JS_LeaveCompartment